README.md

```diff
@@ -287,53 +287,161 @@ Although this example is pretty simple and not even includes Lens composition, i
 
 Just as with React, but assuming Angular instead :)
 
+The code for below example - can be found here <https://github.com/georgebatalinski/telescope_todo>
+
 As a bootstrap for the application we need something like:
 
+```typescript 
+@NgModule({
+  declarations: [
+    AppComponent,
+    TodoListComponent
+  ],
+  imports: [
+    ...
+  ],
+  providers: [
+    { provide: 'StreamContract', useFactory: ()=> Telescope.of( AppState.empty() ), deps: [] }
+  ],
+  bootstrap: [AppComponent]
+})
+export class AppModule { }
+```
+
+We define our global STATE (however, the more ultimate approach would be to have local state)
+local state = would mean each Service, would have a Telescope instance of their own, resulting in us not 
+having to setup and/or maintain <AppState> 
+Note: If choose to maintain <AppState> as in this example, we would be adding more members to the 
+class, which would mean boilerplate. 
+
+
+```typescript
+interface STATE {
+    todos: TodoState;
+    people: PersonsState;
+}
+
+
+export class AppState implements STATE {
+
+    static empty() {
+        return new AppState( 
+            new TodoState(), 
+            new PersonsState() 
+            );
+    }
+    todos: TodoState;
+    people: PersonsState;
+    
+    constructor(todos: TodoState, people: PersonsState ) {
+        this.todos = todos;
+        this.people = people;
+    }
+}
+
+
+export class TodoState {
+    todos: Array<Todo>;
+
+    constructor( todos: Array<Todo> = [] ) {
+        this.todos = todos;
+    }
+
+    add(title: string): TodoState {
+        let todos = [ {title } ];
```

Comment:
You are creating new states with this operation, consider using const for all variables when possible. Although using `const` can be considered a style preference, it is a preference I'd rather keep in this examples for consistency.